elixir/apps/web/lib/web/live/resources/components.ex

```diff
@@ -206,14 +206,14 @@ defmodule Web.Resources.Components do
 
           <div class="mt-2.5 w-24">
             <.input
-              title="Allow ICMP traffic"
+              title="Allow ICMP echo requests/replies"
               type="checkbox"
               field={@forms_by_protocol[:icmp]}
               name={"#{@form.name}[icmp][enabled]"}
               checked={Map.has_key?(@forms_by_protocol, :icmp)}
               value="true"
               disabled={!@traffic_filters_enabled?}
-              label="ICMP"
+              label="ICMP echo"
```

Comment:
@jamilbk Just making sure you've seen these changes in here?

Reply:
Yeah thanks for adding that. I think that's probably a good way to summarize the effect this change has.